src/StatsigUser.ts

```diff
@@ -1,7 +1,9 @@
 import { StatsigEnvironment } from './StatsigOptions';
 
-export type StatsigUser = {
-  userID?: string;
+export type StatsigUser = ( // at least one of userID or customIDs must be provided
+  | { userID: string }
+  | { customIDs: Record<string, string> }
+) & {
```

Comment:
you can provide both userID and customIDs, but I was just testing it out and it looks like this type works for that case as well!

Reply:
yeah I guess my code comment could have been more clear that "or" was intended to mean logical or, rather than how we normally think of "or" as meaning "one or the other but not both". but at least the type works the way we wanted it!